mgosimplekv/kv.go

```diff
@@ -0,0 +1,166 @@
+package mgosimplekv
```

Comment:
Licence ? In general licence in inconsistent and date also.

Reply:
done

Reply:
it doesn't look very done

Reply:
ah yes, done now

Reply:
I changed the TODO.